### PR TITLE
PLAT-132612: Fix SpotlightRootDecorator to show focus effect when enter key is pressed in touch mode

### PR DESCRIPTION
**Pull request by changgilee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
When press enter in touch mode, focus is not restored

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Change touch state when enter key is occur in touch mode

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
N/A

### Links
[//]: # (Related issues, references)
PLAT-132612

### Comments
N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `spotlight/SpotlightRootDecorator` to change non touch mode when `enter` key is down in touch mode
```
Comment (MikyungKim) [inline CHANGELOG.md:9]: Please write the changelog under packages/spotlight/CHANGELOG.md.

**Review by jeonghee27 — APPROVED** [review]
LGTM except for ChangeLog.
MK will review ChangeLog

File: CHANGELOG.md (hunk 1 above)
Comment (MikyungKim) [reply CHANGELOG.md:9]:
```suggestion
- `spotlight/SpotlightRootDecorator` to show focus effect when `enter` key is pressed in touch mode
```

Comment (changgilee) [reply CHANGELOG.md]: Fix it. Thanks~

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**